python/mxnet/kvstore.py

```diff
@@ -349,6 +349,88 @@ def row_sparse_pull(self, key, out=None, priority=0, row_ids=None):
             check_call(_LIB.MXKVStorePullRowSparse(
                 self.handle, mx_uint(len(ckeys)), ckeys, cvals, crow_ids, ctypes.c_int(priority)))
 
+    def set_gradient_compression(self, compression_params=None):
```

Comment:
What's the point of having None as default value?

Reply:
Ok, will change to 2bit